src/components/Exercise.stories.tsx

```diff
@@ -125,20 +126,24 @@ const exerciseWithQuestionStatesProps: ExerciseWithQuestionStatesProps = {
       attempt_number: 0,
       incorrectAnswerId: 0,
       canAnswer: true,
-      needsSaved: true
+      needsSaved: true,
+      apiIsPending: false
     }
   },
 };
 
 export const Default = () => {
-  const [selectedAnswerId, setSelectedAnswerId] = useState<ID>(0);
+  const [selectedAnswerId, setSelectedAnswerId] = useState<number>(0);
   const [apiIsPending, setApiIsPending] = useState(false)
   exerciseWithQuestionStatesProps.questionStates['1'].answer_id = selectedAnswerId;
-  exerciseWithQuestionStatesProps.apiIsPending = apiIsPending;
+  exerciseWithQuestionStatesProps.questionStates['1'].apiIsPending = apiIsPending;
   return (
     <Exercise
       {...exerciseWithQuestionStatesProps}
-      onAnswerChange={(a) => setSelectedAnswerId(a.id)}
+      onAnswerChange={(a: Omit<Answer, 'id'> & { id: number, question_id: number }) => {
+        console.log('question_id', a.question_id, 'answer_id', a.id);
```

Comment:
just flagging in case you meant to remove

Reply:
Haha yeah I couldn't decide if we'd find it helpful for testing or not, but probably better to remove.